stack_array.py

```diff
@@ -0,0 +1,9 @@
+import numpy as np
+
+
+def convert_arrays(array1, array2):
+    return np.dstack((array1, array2))
```

Comment:
You're not returning an iterator- look at the exercise's specifications for what this function should return.